etest/qemu.py

```diff
@@ -0,0 +1,34 @@
+"""QEMU interpreter management."""
+# Copyright (C) 2014 by Alex Brandt <alunduil@alunduil.com>
+#
+# etest is freely distributable under the terms of an MIT-style license.
+# See COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from etest import docker
+
+
+class qemu:
+    """QEMU interpreter management."""
+
+    def __init__(self, arch: str):
+        """Initialize."""
+        if arch != "amd64":
+            self.enabled = True
```

Comment:
This seems weird being a public property that I would read as enabling or disabling QEMU.  Should we instead make it _use?

Reply:
If by _use you mean make it use __ then that seens sensible

Reply:
No, I mean `_use` indicating it's private to the type rather than part of the public interface.

Reply:
I cant find reference on this. Could you send some? Sorry

Reply:
https://pep8.org/#method-names-and-instance-variables